src/sbt-test/sbt-compatibility/library/test

```diff
@@ -0,0 +1,47 @@
+> v0-1-0/publishLocal
+> reload
+
+# NOT OK to introduce source or binary incompatibilities
+# during 0.1.x
+-> v0-1-1/mimaReportBinaryIssues
+-> v0-1-1/compatibilityCheck
+> v0-1-1/publishLocal
```

Comment:
Should we skip publishing this one since it violates the versioning policy?

Reply:
It'd be better this way, but that breaks what we test here. (If we don't publish the versions breaking bin compat, we don't have the expected compatibility at the next version.)

Reply:
The sources of 0.2.0 look wrong to me, they should be the same as 0.1.1. I can make another PR to fix the tests, if you want.

Reply:
But still, this test scenario is wrong, we should not have to publish 0.1.1. Checking compat on 0.2.0 should work also if only 0.1.0 was published.

Reply:
See #25